src/test/java/org/elasticsearch/index/mapper/update/default_mapping_with_disabled_root_types.json

```diff
@@ -1 +1 @@
-{"type":{"_timestamp":{"enabled":false},"_index":{"enabled":false},"_size":{"enabled":false},"properties":{}}}
\ No newline at end of file
+{"type":{"_timestamp":{"enabled":false},"_index":{"enabled":false},"_size":{"enabled":false}}}
```

Comment:
Yes, so that tests can check `"foo".equals(string)` instead of `"foo\n".equals(string)`